README.md

```diff
@@ -9,3 +9,664 @@ ExpressRouterAdapter is available as a package on NPM for use with a module bund
 ```shell
 npm install --save @symbiotic/express-router-adapter
 ```
+
+## Getting started
+
+ExpressRouterAdapter enhances express, so lets start with a simple express app. This is not a deep dive intro express, see express documentation.
+
+```js
+
+const app = express();
+
+app.get('/pets/:petId', (req, res) => {
+  res.json({ petId: req.params.petId, name: 'Sawyer '});
+});
+
+app.listen(4000);
+
+```
+
+Lets install ExpressRouterAdapter and dependncies
+
+```shell
+npm install --save @symbiotic/express-router-adapter aurelia-dependency-injection reflect-metadata
+```
+
+We are using IOC in our examples but you could new up all the classes yourself if you choose, in which case you would not need `aurelia-dependency-injection` and `reflect-metadata`.
+
+Now we can add ExpressRouterAdapter and refactor to use RouteMetaBuilder
+
+```js
+import 'reflect-metadata'; // polyfill for aurelia-dependency-injection
+import express from 'express';
+import { ExpressRouterAdapter, RouterMetaBuilder, RouteProvider } from '@symbiotic/express-router-adapter';
+import { Container } from 'aurelia-dependency-injection';
+
+const container = new Container();
+
+class PetRouter{
```

Comment:
Consider adding a comment 'Routers must implement a getRoutes method that returns an array of IHTTPRoute (usually built by RouterMetaBuilder)'